src/app/calendar/_components/timetable/TimeTableMainPart.tsx

```diff
@@ -102,67 +109,26 @@ export function CalendarTimeTable({ reservations }: CalendarTimeTableProps) {
   };
 
   const timestampsMap = mapWeekdaysToTimestamps(weekdaysMap(reservations));
-
   return (
-    <div className="relative">
-      <div className="grid-col-9 grid w-full grid-rows-1">
-        <div className="border-r-4 border-black p-2">
-          <div className="border-b-4 border-black pb-3 text-center">Time</div>
-          <div className="pt-2">
-            {Array.from({ length: 15 }, (_, i) => (
-              <React.Fragment key={i}>
-                <div className="h-2.5 border-t-2 border-black">{i + 7}:00</div>
-                <div className="h-2.5"></div>
-                <div className="h-2.5"></div>
-                <div className="h-2.5"></div>
-              </React.Fragment>
-            ))}
-          </div>
+    <React.Fragment>
```

Comment:
If you don't put a key on React.Fragment (like in a loop) you can just use empty brackets
```suggestion
    <>
```